scraper.rb

```diff
@@ -115,6 +115,37 @@ def name_with_title
   SearchPage.new(response: Scraped::Request.new(url: url).response).members
 end.uniq
 
+# The search index and list pages seem to be missing some members of
+# the National Assembly, so add them manually to the pages to be scraped:
+members.concat [
```

Comment:
`.concat` here will simply add these onto the end, even if they're already there (i.e. should the search start returning them again), which doesn't seem ideal. It's not a massive issue, but that would involve not only re-fetching, re-parsing, and re-saving the data, but also pushing it to the scraped page archive again, so it would be good to avoid that.

As you don't actually need the `name` part here, as the loop is only dealing with the URL, I'd be tempted to first build the list of URLs from the `SearchPage`, then `|` this second list with it, and then do the fetch loop over that.